src/analysis/sensitivity.jl

```diff
@@ -440,8 +468,10 @@ function rsa(X::DataFrame, y::AbstractVector{<:Real}; S::Int64=10)::NamedDimsArr
         NamedDimsArray(r_s; bins=string.(seq[2:end]), factors=Symbol.(names(X)))
     )
 end
-function rsa(rs::ResultSet, y::AbstractVector{<:Real}; S::Int64=10)::NamedDimsArray
-    return rsa(rs.inputs, vec(y); S=S)
+function rsa(
+    rs::ResultSet, y::AbstractVector{<:Real}, model_spec::DataFrame; S::Int64=10
+)::NamedDimsArray
+    return rsa(rs.inputs, vec(y), model_spec; S=S)
 end
```

Comment:
`rs` holds a copy of model spec, so I'm not sure why we need it as an argument here.

```suggestion
function rsa(
    rs::ResultSet, y::AbstractVector{<:Real}; S::Int64=10
)::NamedDimsArray
    return rsa(rs.inputs, vec(y), rs.model_spec; S=S)
end
```